frontend/src/e2e-test/pages/employee/holiday-periods.ts

```diff
@@ -89,19 +104,20 @@ export class HolidayPeriodsPage {
     descriptionLink?: string
     descriptionLinkSv?: string
     descriptionLinkEn?: string
-    activeStart?: string
-    activeEnd?: string
+    activeRange?: FiniteDateRange
     fixedPeriodOptions?: string
     fixedPeriodOptionLabel?: string
   }) {
     for (const [key, val] of Object.entries(params)) {
       if (val !== undefined) {
         if (key === 'period') {
```

Comment:
`params` doesn't have `period`, so is this special handling needed? 🤔

Reply:
I don't think it's needed anymore, the field was removed in 4bfa5920cb1fade8b6ac32a15742c61ff656ccce.